laws/applicative.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+const {identity, apply} = require('fantasy-combinators');
+const {ap} = require('../index');
+
+const id = (t) => (eq) => (x) => {
+    const a = t.of(identity).ap(t.of(x));
```

Comment:
This should be `t[of](identity)[ap](t[of](x))` I think. 

Btw we have a problem with `of` here, it's a reserved word, we can't use it for variable name :crying_cat_face: